cetloxblock/cetloxblock_xblock.egg-info/entry_points.txt

```diff
@@ -0,0 +1,3 @@
+[xblock.v1]
```

Comment:
We need to add a `.gitignore` file to ignore these things in version control -- they shouldn't be in the public repository.

The same goes for `.pyc` files, which seemed to have gotten checked in.

Here's an example you can take and drop in the root of this repo: https://github.com/open-craft/xblock-poll/blob/master/.gitignore

Reply:
Also, since I'm here, I should say we need quality checks on this code to keep to a standard as we develop.

For this, we'll need a `requirements.txt` file, similar to what you see [here](https://github.com/open-craft/problem-builder/blob/master/requirements.txt). (Only include what we need, of course). We can run `pylint` or `flake8`, whatever you prefer, but that'll need to be added to the `requirements.txt` file, and pinned to a version.

Then we can enforce that quality checks using that tool pass before merging any new PRs in the future.

Reply:
Sorry, I need help with this comment. What should I have in requirements.txt? my single .py file imports only pkg_resources, and setup.py imports os. Should they be there? I am not familiar with python and its testing tools. I install my xblock to devstack without requirements.txt.